### DISCUSSION TITLE
How to get PowerManager.SystemSuspendStatus when status change?

### DISCUSSION THREAD

**Discussion by Maroet** [microsoft/WindowsAppSDK#1924] [Q&A]
When I try to get PowerManager.SystemSuspendStatus in status changed event handler like below, I got an error "Error HRESULE E_FAIL has been returned from a call to a COM component.". 
I can get some other powermanager status like UserPrensenceStatus, ScreenStatus, only SystemSuspendStatus has such a problem in this context. 

        public MainWindow()
        {
            this.InitializeComponent();
            PowerManager.SystemSuspendStatusChanged += PowerManager_SystemSuspendStatusChanged;
        }

        private void PowerManager_SystemSuspendStatusChanged(object sender, object e)
        {
            try
            {
                _Log.Information(PowerManager.SystemSuspendStatus.ToString());
            }catch(Exception ex)
            {
                _Log.Information(ex.Message);
            }
        }
My question is, is this really a known problem or is the way I use it incorrect?

**Comment by Wielding** [comment]
I have the same problem almost a year after this post with no responses.

**Comment by codendone** [comment]
The PowerManager code is broken. Closing this discussion to track with this issue: #2833
